### PR TITLE
update sqltoolsservice for dacfx

### PR DESCRIPTION
**Pull request by udeeshagautam**
For a few bugs across Schema Compare and project extensions.

### CODE REVIEW

**Comment by udeeshagautam** [comment]
@Charles-Gagnon @alanrenmsft @kburtram  - seems like sqltools service was not updated in ADS for a bit - wanted to confirm that there is no to no do it. I validated basic connection and Dacfx scenarios and things look good.

**Comment by Charles-Gagnon** [comment]
I don't see anything hugely concerning

@alanrenmsft @Benjin @cssuh - any issues from the stuff you all did/were involved in?

**Review by Charles-Gagnon — APPROVED** [review]

**Review by kisantia — APPROVED** [review]

File: extensions/mssql/config.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/v{#version#}/microsoft.sqltools.servicelayer-{#fileName#}",
-	"version": "3.0.0-release.68",
+	"version": "v3.0.0-release.81",
```
Comment (alanrenmsft) [inline extensions/mssql/config.json:3]: is the leading character 'v' intentionally added?

Comment (udeeshagautam) [reply extensions/mssql/config.json:3]: absolutely not, thanks for the save! missed the fixing commit.

**Review by alanrenmsft — APPROVED** [review]

**Pull request merged**